spec/helpers/previous_url_helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+class PreviousUrlHelper::DummyView
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.